JavaScript/JavaScriptSDK/ajax/ajaxUtils.ts

```diff
@@ -31,17 +31,17 @@ module Microsoft.ApplicationInsights {
 
     export class dateTime {
         ///<summary>Return the number of milliseconds since 1970/01/01 in local timezon</summary>
-        public static Now = (window.performance && window.performance.now) ?
+        public static Now = (window.performance && window.performance.now && window.performance.timing) ?
             function () {
-                return performance.now();
+                return window.performance.now() + window.performance.timing.navigationStart;
```

Comment:
This is probably fine. :-) Be aware that mixing this and Date.now() can cause variance of up to ~16 ms as they are not guaranteed to use the same internals for calculating their times. Again.. probably fine.